README.md

```diff
@@ -1,38 +1,53 @@
 # ctb-backend
 
-Jak to uruchomić?
+## How to run
 
-Najpierw w celu zbudowania obrazu wykonaj polecenie:
+At the beginig you must create an image:
 ```
 docker build . -t ctb
 ```
 
-Następnie w celu uruchomienia kontenera:
+The next step is starting container. For that you must type:
```

Comment:
"The next step is starting container..." -> "Start a container:"

Try to keep it imperative.